Autopilot/PathManager/Src/trackFinder.cpp

```diff
@@ -0,0 +1,18 @@
+#include <math.h>
+#include "trackFinder.hpp"
+
+double getTrack(double latitudeInitial, double longitudeInitial, double latitudeCurrent, double longitudeCurrent)
```

Comment:
I'm not fond of this prototype because you're asking the caller to remember past datapoints for you. Better is to just have them use the most recent data point. You can store the past datapoints in a static array or something inside the function. Also, this way, you can modify the precision of your algorithm (how many terms you use and all) without asking the caller to change his method of calling.